### PR TITLE
TRITON-2190 sdc-network list should exclude fabric networks by default

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

File: tools/bin/sdc-network [hunk 1]
```diff
@@ -38,19 +38,24 @@ Usage:
...
+    fabric='false'
+    if [[ $1 == '-f' ]]; then
+        fabric='true'
+    fi
+    printf "%-40s %-36s  %-4s %16s %16s\n" \
         "NAME" "UUID" "VLAN" "SUBNET" "GATEWAY"
     # BEGIN BASHSTYLED
-    printf "%-12s %36s  %4s %16s %16s\n" \
-        $(napi /networks \
+    printf "%-40s %36s  %4s %16s %16s\n" \
```
Comment (bahamat) [inline tools/bin/sdc-network:57]: Made this wider to accommodate some of our network names

**Comment by bahamat** [comment]
Tested by putting it on a headnode and running it.

**Review by twhiteman — DISMISSED** [review]
LGTM.

**Review by twhiteman — APPROVED** [review]

**Review by twhiteman — APPROVED** [review]
2021 YAY!

**Pull request merged**